roles/azure_manage_postgresql/defaults/main.yml

```diff
@@ -1,8 +1,3 @@
 ---
 azure_manage_postgresql_operation: create
-azure_manage_postgresql_postgresql_version: "9.5"
-azure_manage_postgresql_postgresql_settings: []
-azure_manage_postgresql_postgresql_firewall_rules: []
-azure_manage_postgresql_postgresql_database_instances: []
-azure_manage_postgresql_delete_server: false
-azure_manage_postgresql_delete_resource_group: false
```

Comment:
I think it's fine to leave it here—azure_manage_postgresql_delete_resource_group. We didn't define it in the default file of the tests. In all other roles this var in the default of the role.

Reply:
In the case of the:
azure_manage_postgresql_delete_server: false
and
azure_manage_postgresql_delete_resource_group: false

We added a condition in the role that states that if the variable does not exists, it will handle it like "false" so the variable in the defaults file is redundant.

Reply:
From what I understood, statement will only be executed if:

The variable azure_manage_postgresql_delete_resource_group is defined.
The value of azure_manage_postgresql_delete_resource_group is true or evaluates to true.